queries.sql

```diff
@@ -1,3 +1,44 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
\ No newline at end of file
+SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth < '2019-01-01';
+SELECT name FROM animals WHERE neutered = 't' AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR 'Pikachu';
+SELECT  name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered = 'true';
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg > 10.4 AND weight_kg =< 17.3;
+
+/* task-2 */
+BEGIN;
+   UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
* I suggest you properly state these insertions `Inside a transaction update the animal's table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.`  
* You only started the transaction, made the change (` UPDATE animals SET species = 'unspecified';`) but you didn't verify that change was made and undone for example:  ` SELECT species from animals;` 
  ` ROLLBACK;`
Then
  ` SELECT species from animals; `
`

Reply:
Thanks a lot for your explanation. It was really helpful. :+1: